docs/self-hosted-appcircle/install-server/helm-chart/installation/docker-k8s-migration.md

```diff
@@ -0,0 +1,1343 @@
+
+  ```bash
+  cat projects/spacetech/user-secret | base64 -d
+  ```
+
+- **Generated Secrets:**
+  Print the `generated-secret.yaml` file and save it on the bastion host:
+
+  ```bash
+  cat projects/spacetech/generated-secret.yaml
+  ```
+
+- **`cred.json` file:**
+  Print the `cred.json` file and save it on the bastion host:
+  ```bash
+  cat cred.json
+  ```
+
+#### 3. Check the Data Size on the Standalone Appcircle Server
+
+- **Get the volume sizes:**
+
+  ```bash
+  export APPCIRCLE_DISK_USAGE=$(docker system df -v)
+  ```
+
+- **Check the MinIO data size:**
+
+  ```bash
+  echo "$APPCIRCLE_DISK_USAGE" | grep "spacetech_snsd_data"
+  ```
+
+- **Check the MongoDB data size:**
+
+  ```bash
+  echo "$APPCIRCLE_DISK_USAGE" | grep "spacetech_mongo_data"
+  ```
+
+- **Check the PostgreSQL data size:**
+
+  ```bash
+  echo "$APPCIRCLE_DISK_USAGE" | grep "spacetech_posgresqlData"
+  ```
+
+- **Check the Vault data size:**
+  ```bash
+  echo "$APPCIRCLE_DISK_USAGE" | grep "spacetech_vault_data"
+  ```
+
+You will use those values while configuring the `values.yaml` of the Appcircle server Helm chart.
+
+#### 4. Make Sure the Standalone Appcircle Server Is Running
+
+Before proceeding with the migration, verify that the standalone Appcircle server is operational and healthy. This ensures you can create accurate backups without issues.
+
+- **Log in to the standalone Appcircle server:**
+
+- **Change directory to the `appcircle-server`:**
+
+   ```bash
+   cd appcircle-server
+   ```
+
+- **Check the health status of the standalone Appcircle server:**
+   <SpacetechExampleInfo/>
+
+   ```bash
+   ./ac-self-hosted.sh -n spacetech check
+   ```
+
+#### 5. Update the Standalone Appcircle Server to the Latest Version
+
+Before migrating to Kubernetes Appcircle server, please [update the standalone Appcircle server](https://docs.appcircle.io/self-hosted-appcircle/install-server/linux-package/update) to the latest version.
+
+#### 6. Stop the Standalone Appcircle Server Requests.
+
+For the migration, there should be no running builds on the Appcircle during the migration. Also prevent external requests while keeping the server healthy by stopping the Nginx service.
+
+- **Log in to the standalone Appcircle server:**
+
+- **Change directory to the `appcircle-server`:**
+
+   ```bash
+   cd appcircle-server
+   ```
+
+- **Change directory to the `export` directory of the project:**
+   <SpacetechExampleInfo/>
+   ```bash
+   cd projects/spacetech/export
+   ```
+
+- **Stop the Nginx service:**
+   ```bash
+   docker compose stop nginx
+   ```
+
+## Installation
+
+### 1. Create `values.yaml`
+
+Below is a minimal `values.yaml` file that you should configure for your deployment.
+
+**Please adjust these values** according to your environment requirements and **save your file**.
+
+In the example values below, we used `spacetech` as an **example organization name**. You should **replace it** with your actual organization name or any other value you prefer.
```

Comment:
We should add a "danger" box that tells user not to change any setting in global yaml and use them as-is.